src/pylint_odoo/checkers/__init__.py

```diff
@@ -1,3 +1,4 @@
 from . import odoo_addons
 from . import vim_comment
 from . import custom_logging
+from . import import_checker
```

Comment:
Why a new file? Shouldn't you implement new lints into the existing file for Odoo addons?

@moylop260 I don't have enough context to know if this is correct.

Reply:
I think odoo_addons is getting too big and grouping lints inside different checkers could help organization.